galley/pkg/kube/source/monitoring.go

```diff
@@ -88,6 +89,9 @@ func recordConverterResult(success bool, apiVersion, group, kind string) {
 	} else {
 		metric = sourceConversionFailure
 	}
+	if len(group) == 0 {
+		group = coreGroup
+	}
```

Comment:
lgtm

Can you add a short one-line comment about why this is needed for future readers?

Reply:
Thanks for the feedback.
I added a comment, I hope it makes things clearer.